README.md

```diff
@@ -12,9 +12,10 @@
 # It's recommanded to use virtual environment
 python -m pip install --upgrade virtualenv
 virtualenv .venv
```

Comment:
Not related to your change per-se but python has `venv` built in now since 3.3 - We should just use that :D

Reply:
Sure, please update the instructions. 👌

Reply:
```suggestion
python -m venv .venv
```